### PR TITLE
Refresh camera image on ROS network reconnection

### PR DESCRIPTION
**Pull request by Bitterisland6**
Added functionalities to perform automatic reconnection to ROS webbridge server and update of camera view and topic listing.
Also added option to choose empty stream - no image.

### CODE REVIEW

File: js/leo.js [hunk 1]
```diff
@@ -18,6 +18,12 @@ var topicsForTypeClient;
 
 var select;
 
+var intervalFlag = false;
+var initROSinterval;
+var last_selection;
```
Comment (bjsowa) [inline js/leo.js]: Don't mix up different naming cases for variables
```suggestion
var lastSelection;
```

File: js/leo.js [hunk 2]
```diff
@@ -245,20 +299,26 @@ function imgWidth() {
...
 
     robot_hostname = location.hostname;
 
     initROS();
     initTeleopKeyboard();
     createJoystick();
-    getVideoTopics();
 
     video = document.getElementById('video');
     select = document.getElementById('camera-select');
```
Comment (bjsowa) [inline js/leo.js]: possible race condition, retrieve these elements before call to `initROS`

**Review by bjsowa — CHANGES_REQUESTED** [review]

File: js/leo.js (hunk 1 above)
Comment (Bitterisland6) [reply js/leo.js]: Done for all variables.

File: js/leo.js (hunk 2 above)
Comment (Bitterisland6) [reply js/leo.js]: Done.

**Review by bjsowa — APPROVED** [review]

**Pull request merged**